src/dbus/client/thread_api_dbus.cpp

```diff
@@ -350,78 +350,148 @@ OtbrClientError ThreadApiDBus::RemoveOnMeshPrefix(const otbrIp6Prefix &aPrefix)
 
 OtbrClientError ThreadApiDBus::SetMeshLocalPrefix(const std::array<uint8_t, OTBR_IP6_PREFIX_SIZE> &aPrefix)
 {
-    return SetProperty(OTBR_DBUS_MESH_LOCAL_PREFIX_PROPERTY, aPrefix);
+    return SetProperty(OTBR_DBUS_PROPERTY_MESH_LOCAL_PREFIX, aPrefix);
 }
 
 OtbrClientError ThreadApiDBus::SetLegacyUlaPrefix(const std::array<uint8_t, OTBR_IP6_PREFIX_SIZE> &aPrefix)
 {
-    return SetProperty(OTBR_DBUS_LEGACY_ULA_PREFIX_PROPERTY, aPrefix);
+    return SetProperty(OTBR_DBUS_PROPERTY_LEGACY_ULA_PREFIX, aPrefix);
 }
 
 OtbrClientError ThreadApiDBus::SetLinkMode(const otbrLinkModeConfig &aConfig)
 {
-    return SetProperty(OTBR_DBUS_LINK_MODE_PROPERTY, aConfig);
+    return SetProperty(OTBR_DBUS_PROPERTY_LINK_MODE, aConfig);
 }
 
 OtbrClientError ThreadApiDBus::GetLinkMode(otbrLinkModeConfig &aConfig)
 {
-    return GetProperty(OTBR_DBUS_LINK_MODE_PROPERTY, aConfig);
+    return GetProperty(OTBR_DBUS_PROPERTY_LINK_MODE, aConfig);
 }
 
 OtbrClientError ThreadApiDBus::GetDeviceRole(otbrDeviceRole &aRole)
 {
     std::string     roleName;
     OtbrClientError err;
 
-    SuccessOrExit(err = GetProperty(OTBR_DBUS_DEVICE_ROLE_PROPERTY, roleName));
+    SuccessOrExit(err = GetProperty(OTBR_DBUS_PROPERTY_DEVICE_ROLE, roleName));
     SuccessOrExit(err = NameToDeviceRole(roleName, aRole));
 exit:
     return err;
 }
 
 OtbrClientError ThreadApiDBus::GetNetworkName(std::string &aNetworkName)
 {
-    return GetProperty(OTBR_DBUS_NETWORK_NAME_PROPERTY, aNetworkName);
+    return GetProperty(OTBR_DBUS_PROPERTY_NETWORK_NAME, aNetworkName);
 }
 
 OtbrClientError ThreadApiDBus::GetPanId(uint16_t &aPanId)
 {
-    return GetProperty(OTBR_DBUS_PANID_PROPERTY, aPanId);
+    return GetProperty(OTBR_DBUS_PROPERTY_PANID, aPanId);
 }
 
 OtbrClientError ThreadApiDBus::GetExtPanId(uint64_t &aExtPanId)
 {
-    return GetProperty(OTBR_DBUS_EXTPANID_PROPERTY, aExtPanId);
+    return GetProperty(OTBR_DBUS_PROPERTY_EXTPANID, aExtPanId);
 }
 
 OtbrClientError ThreadApiDBus::GetChannel(uint16_t &aChannel)
 {
-    return GetProperty(OTBR_DBUS_CHANNEL_PROPERTY, aChannel);
+    return GetProperty(OTBR_DBUS_PROPERTY_CHANNEL, aChannel);
 }
 
 OtbrClientError ThreadApiDBus::GetMasterKey(std::vector<uint8_t> &aMasterKey)
```

Comment:
As c++ API, could we remove the prefix `OtbrClient`, just `Error`?

Reply:
Calling it `Error` looks too generic, I'll call it `ClientError`.